README.adoc

```diff
@@ -1,129 +1,104 @@
-= Mobile Docs
+= Documentation for AeroGear Mobile Services 
 
-Our goal is to help users of AeroGear projects "understand things well and not feel like they're dealing with a capricious minor god instead of a product." footnote:[Laura Bailey]
+This readme contains the guidelines and standards for contributing to AeroGear documentation. The documentation is written in link:https://asciidoctor.org/docs/what-is-asciidoc/[AsciiDoc^] and published by link:docs.antora.org[Antora^].
 
-This readme introduces the standards for AeroGear docs.
+- If you see an issue in documentation, link:https://github.com/aerogear/mobile-docs/issues/new[raise an issue^] and post a PR with a fix for a review. Please include as much information as possible.
+- See the link:https://github.com/aerogear/mobile-docs/projects/1[Kanban board with Docs Issues^].
 
 :toc: 1
 
-== Concepts, Procedures and Reference
+== Writing AeroGear Documentation
```

Comment:
the order of the items here is not the order that we want to communicate to contributors

Reply:
what do you mean?